docs/DEVELOPING.md

```diff
@@ -343,6 +343,114 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## Microsoft SQL Connector
+
+The [Elastic Microsoft SQL connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/mssql.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+**Prerequisite**
+
+The Microsoft SQL connector requires an ODBC driver to execute. To install ODBC driver please refer following [documentation](https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-driver-manager).
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Microsoft SQL details](#gather-microsoft-sql-details)
+2. [Configure Microsoft SQL connector](#configure-microsoft-sql-connector)
+
+#### Gather Microsoft SQL details
+
+Collect the information that is required to connect to your Microsoft SQL Server:
+
+- The server host address where the Microsoft SQL Server is hosted.
+- The port where the Microsoft SQL server is hosted.
+- The username the connector will use to log in to Microsoft SQL.
+- The password the connector will use to log in to Microsoft SQL.
+- The Microsoft SQL database name where the connector will crawl to fetch rows.
+
+#### Configure Microsoft SQL connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `host`
+
+The server host address where the Microsoft SQL Server is hosted. Default value is `127.0.0.1`. Examples:
+
+  - `192.158.1.38`
+  - `demo.instance.demo-region.demo.service.com`
+
+##### `port`
+
+The server port where Microsoft SQL Server is available. Default value is `9090`. Examples:
+
+  - `5432`
+  - `9090`
+
+##### `user`
```

Comment:
IMO, `username` is a more intuitive option. ANd I see it's used for Network Drive Connector.

Reply:
Done

Reply:
I think it's ok to leave the `key` of the configuration as `user` if it's already implemented. Just make sure to use `username` in the label and description, i.e. whatever is displayed to customers.

Reply:
Yes, the key is `user` But the label was `username`, So we can add `username` in the doc also.

Reply:
The documentation is used for connector clients so we should be as exact as possible when referencing config parameters. 
Personally, I find this confusing. Are we using **keys** or **labels** in the documentation?
I don't have a strong opinion on which should be used, as long as we're consistent. I was always assuming you referenced keys in the DEVELOPING.md doc.
While most are identical, there are several parameters that use different keys.

- Keys: "host" / "port" /  "**user**" / "password" / "database" / "**tables**" / "**fetch_size**" / "**retry_count**"
- Labels: "Host" / "Port" / "**Username**" / "Password"/ "Database" / "**Comma-separated list of tables**" / "**Rows fetched per request**" / "**Retries per request**"

I see several options:
1. Rename the key for DB generic connector to `username`, that would make it consistent to other connector clients. Any reason to use "user" for DB connector while using the "username" key for Network Drive? 
2. Consistently use labels for all fields of connector client if that's the preferred approach.

My vote is for option 1.

Reply:
**++** for consistency.

Reply:
Change key user to username in #565